Cargo.lock

```diff
@@ -730,13 +804,16 @@ dependencies = [
 
 [[package]]
 name = "crates-index-diff"
-version = "10.0.0"
+version = "11.0.0"
```

Comment:
this will probably change to `11.1.1`, right? :)

Reply:
(I saw the bug-reports & new releases )

Reply:
Indeed, there were a few improvements and the version number was updated in e5320eb8.